ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/commons/objective_function_evaluator/CostEvaluator.java

```diff
@@ -33,24 +34,11 @@ public interface CostEvaluator {
      * @param rangeActionActivationResult
      * @param sensitivityStatus : the sensitivity computation status
      */
-    double computeCost(FlowResult flowResult, RangeActionActivationResult rangeActionActivationResult, SensitivityResult sensitivityResult, ComputationStatus sensitivityStatus);
+    Pair<Double, List<FlowCnec>> computeCostAndLimitingElements(FlowResult flowResult, RangeActionActivationResult rangeActionActivationResult, SensitivityResult sensitivityResult, ComputationStatus sensitivityStatus);
```

Comment:
add a default implementation to call the next function with an empty contingenciesToExclude, and remove the implementation from the implementing classes